frontend/src/components/ParkingPermitForm.tsx

```diff
@@ -120,12 +125,27 @@ function ParkingPermitForm() {
                     className="w-full p-2 rounded-md bg-neutral-800 text-orange-100"
                 />
             </div>
-
             <div>
                 <p className="w-full p-2 rounded-md bg-neutral-800 text-orange-100">
-                    Valid Until: {new Date(Date.now() + 7 * 24 * 60 * 60 * 1000).toLocaleDateString()}
+                    Valid Until:{" "}
+                    {new Date(Date.now() + 7 * 24 * 60 * 60 * 1000).toLocaleDateString()}
```

Comment:
We don't need to do math.
```suggestion
                    {new Date(Date.now() + 604800000 ).toLocaleDateString()}
```

Reply:
done